core/src/main/java/org/axonframework/queryhandling/SimpleQueryBus.java

```diff
@@ -290,6 +291,16 @@ public void completeExceptionally(Predicate<SubscriptionQueryMessage<?, ?, ?>> f
                       .forEach(uh -> uh.error(cause));
     }
 
+    /**
+     * Provides the set of currently running subscription queries. Returned set represents current state of active
+     * subscriptions on this bus. Do note that the set won't be updated once the state of active subscription changes.
```

Comment:
This is actually inaccurate. The `keySet()` javadoc mentions:

> The set is backed by the map, so changes to the map are reflected in the set, and vice-versa

Wrapping it in the `unmodifiableSet` prevents the "vice-versa" part, but it does mean that the return `Set` automatically reflects any changes made in the subscriptions.

Reply:
You're right! I completely misinterpreted it.